scm-ui/ui-webapp/src/groups/components/table/GroupMember.tsx

```diff
@@ -59,6 +57,10 @@ export default class GroupMember extends React.Component<Props> {
   render() {
     const { member } = this.props;
     const to = `/user/${member.name}`;
-    return <StyledMember>{this.showName(to, member)}</StyledMember>;
+    return (
+      <StyledMember className={classNames("is-inline-block", "mr-1", "px-3", "py-1")}>
```

Comment:
Is `is-inline-block` an actual bulma class ? I couldnt find it in the docs.

Reply:
And you dont need `classNames` here.

Reply:
This is documented with the [visibility/responsive helpers](https://bulma.io/documentation/helpers/visibility-helpers/).